pandas/tests/io/json/test_pandas.py

```diff
@@ -1217,6 +1227,13 @@ def test_read_json_table_dtype_raises(self, dtype):
         with pytest.raises(ValueError):
             pd.read_json(dfjson, orient='table', dtype=dtype)
 
+    def test_read_json_table_convert_axes_raises(self):
+        # GH25433 GH25435
+        df = DataFrame([[1, 2], [3, 4]], index=[1., 2.], columns=['1.', '2.'])
+        dfjson = df.to_json(orient='table')
+        with pytest.raises(ValueError):
```

Comment:
Can you check the error message using the `match` parameter of `pytest.raises`?  There's an ongoing effort to add this to all tests that check if an error was raised.